client/my-sites/customer-home/style.scss

```diff
@@ -0,0 +1,115 @@
+.customer-home__layout-col {
+	@include breakpoint('>960px') {
+		width: 49%;
+	}
+}
+
+.customer-home__card-subheader {
+	color: var( --color-neutral-light );
```

Comment:
This doesn't pass contrast requirements.

```suggestion
	color: var( --color-text-subtle );
```

Reply:
@davemart-in what do you think?

#### Current, `--color-neutral-light`

<img width="497" alt="Screen Shot 2019-08-09 at 19 59 56" src="https://user-images.githubusercontent.com/1041600/62813457-37e9fc00-bae1-11e9-9855-acc7a726fe2c.png">

#### Suggested, `--color-text-subtle`

<img width="496" alt="Screen Shot 2019-08-09 at 19 59 42" src="https://user-images.githubusercontent.com/1041600/62813458-37e9fc00-bae1-11e9-8d99-2dd25aa15a7d.png">

Reply:
I like the contrast boost!